tests/contest/contest/src/tests/scheduler/scheduler_policy.rs

```diff
@@ -46,11 +46,23 @@ fn scheduler_policy_batch_test() -> TestResult {
 
 pub fn get_scheduler_test() -> TestGroup {
     let mut scheduler_policy_group = TestGroup::new("set_scheduler_policy");
-    let policy_fifo_test = Test::new("policy_other", Box::new(scheduler_policy_other_test));
-    let policy_rr_test = Test::new("policy_batch", Box::new(scheduler_policy_batch_test));
-
-    scheduler_policy_group.add(vec![Box::new(policy_fifo_test)]);
-    scheduler_policy_group.add(vec![Box::new(policy_rr_test)]);
+    let policy_fifo_test = ConditionalTest::new(
+        "policy_other",
+        Box::new(|| match std::env::var("RUNTIME_KIND") {
+            Err(_) => true,
+            Ok(s) => s != "runc",
+        }),
```

Comment:
How about turn into a function and putting others together?

Reply:
others: https://github.com/containers/youki/pull/2647/files#diff-54c4c6b1c0869e602e1d1c3f81dd7162b5ad8e237a6fd7dc477416e27624233cR30-R33

Reply:
Done, please take a look!